<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SDGL691" name="Rediff WebSite Testing" tests="10" failures="1" timestamp="14 Nov 2014 06:20:46 GMT" time="23.048" errors="0">
  <testcase name="@BeforeClass classSetUp" time="0.0" classname="com.rediff.testcases.LoginTest">
    <failure type="com.framework.exception.MyException" message="Error in connecting with Remote WebDriverorg.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &amp;apos;2.42.1&amp;apos;, revision: &amp;apos;68b415adb11acca4286be45124325cc36cfc4a51&amp;apos;, time: &amp;apos;2014-05-29 09:21:55&amp;apos;
System info: host: &amp;apos;SDGL691&amp;apos;, ip: &amp;apos;10.20.20.29&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;x86&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_67&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[com.framework.exception.MyException: Error in connecting with Remote WebDriverorg.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.42.1', revision: '68b415adb11acca4286be45124325cc36cfc4a51', time: '2014-05-29 09:21:55'
System info: host: 'SDGL691', ip: '10.20.20.29', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_67'
Driver info: driver.version: RemoteWebDriver
	at com.framework.testtemplate.TestBase.startWebDriver(TestBase.java:116)
	at com.rediff.testcases.LoginTest.classSetUp(LoginTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @BeforeClass classSetUp -->
  <testcase name="@BeforeClass classSetUp" time="0.0" classname="com.rediff.testcases.HomePageTest">
    <failure type="com.framework.exception.MyException" message="Error in connecting with Remote WebDriverorg.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &amp;apos;2.42.1&amp;apos;, revision: &amp;apos;68b415adb11acca4286be45124325cc36cfc4a51&amp;apos;, time: &amp;apos;2014-05-29 09:21:55&amp;apos;
System info: host: &amp;apos;SDGL691&amp;apos;, ip: &amp;apos;10.20.20.29&amp;apos;, os.name: &amp;apos;Windows 7&amp;apos;, os.arch: &amp;apos;x86&amp;apos;, os.version: &amp;apos;6.1&amp;apos;, java.version: &amp;apos;1.7.0_67&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[com.framework.exception.MyException: Error in connecting with Remote WebDriverorg.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '2.42.1', revision: '68b415adb11acca4286be45124325cc36cfc4a51', time: '2014-05-29 09:21:55'
System info: host: 'SDGL691', ip: '10.20.20.29', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_67'
Driver info: driver.version: RemoteWebDriver
	at com.framework.testtemplate.TestBase.startWebDriver(TestBase.java:116)
	at com.rediff.testcases.HomePageTest.classSetUp(HomePageTest.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @BeforeClass classSetUp -->
  <testcase name="@BeforeMethod LoginTestSetUp" time="0.0" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod LoginTestSetUp -->
  <testcase name="@BeforeMethod HomePageTestSetUp" time="0.0" classname="com.rediff.testcases.HomePageTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod HomePageTestSetUp -->
  <testcase name="@AfterMethod TearDown" time="0.0" classname="com.rediff.testcases.HomePageTest">
    <failure type="com.framework.exception.MyException" message="TestBase -&amp;gt; embedScreenShotIntoReport() - null">
      <![CDATA[com.framework.exception.MyException: TestBase -> embedScreenShotIntoReport() - null
	at com.framework.report.CaptureBrowserScreenShot.embedScreenShotIntoReport(CaptureBrowserScreenShot.java:88)
	at com.framework.testtemplate.TestBase.TearDown(TestBase.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase name="@AfterMethod TearDown" time="0.0" classname="com.rediff.testcases.LoginTest">
    <failure type="com.framework.exception.MyException" message="TestBase -&amp;gt; embedScreenShotIntoReport() - null">
      <![CDATA[com.framework.exception.MyException: TestBase -> embedScreenShotIntoReport() - null
	at com.framework.report.CaptureBrowserScreenShot.embedScreenShotIntoReport(CaptureBrowserScreenShot.java:88)
	at com.framework.testtemplate.TestBase.TearDown(TestBase.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase name="@BeforeMethod LoginTestSetUp" time="0.0" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod LoginTestSetUp -->
  <testcase name="@AfterMethod TearDown" time="0.0" classname="com.rediff.testcases.LoginTest">
    <failure type="com.framework.exception.MyException" message="TestBase -&amp;gt; embedScreenShotIntoReport() - null">
      <![CDATA[com.framework.exception.MyException: TestBase -> embedScreenShotIntoReport() - null
	at com.framework.report.CaptureBrowserScreenShot.embedScreenShotIntoReport(CaptureBrowserScreenShot.java:88)
	at com.framework.testtemplate.TestBase.TearDown(TestBase.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase name="@BeforeMethod LoginTestSetUp" time="0.0" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod LoginTestSetUp -->
  <testcase name="@AfterMethod TearDown" time="0.0" classname="com.rediff.testcases.LoginTest">
    <failure type="com.framework.exception.MyException" message="TestBase -&amp;gt; embedScreenShotIntoReport() - null">
      <![CDATA[com.framework.exception.MyException: TestBase -> embedScreenShotIntoReport() - null
	at com.framework.report.CaptureBrowserScreenShot.embedScreenShotIntoReport(CaptureBrowserScreenShot.java:88)
	at com.framework.testtemplate.TestBase.TearDown(TestBase.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase name="@BeforeMethod LoginTestSetUp" time="0.0" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod LoginTestSetUp -->
  <testcase name="@AfterMethod TearDown" time="0.0" classname="com.rediff.testcases.LoginTest">
    <failure type="com.framework.exception.MyException" message="TestBase -&amp;gt; embedScreenShotIntoReport() - null">
      <![CDATA[com.framework.exception.MyException: TestBase -> embedScreenShotIntoReport() - null
	at com.framework.report.CaptureBrowserScreenShot.embedScreenShotIntoReport(CaptureBrowserScreenShot.java:88)
	at com.framework.testtemplate.TestBase.TearDown(TestBase.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase name="@BeforeMethod LoginTestSetUp" time="0.0" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod LoginTestSetUp -->
  <testcase name="@AfterMethod TearDown" time="0.0" classname="com.rediff.testcases.LoginTest">
    <failure type="com.framework.exception.MyException" message="TestBase -&amp;gt; embedScreenShotIntoReport() - null">
      <![CDATA[com.framework.exception.MyException: TestBase -> embedScreenShotIntoReport() - null
	at com.framework.report.CaptureBrowserScreenShot.embedScreenShotIntoReport(CaptureBrowserScreenShot.java:88)
	at com.framework.testtemplate.TestBase.TearDown(TestBase.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase name="@BeforeMethod LoginTestSetUp" time="0.0" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod LoginTestSetUp -->
  <testcase name="@AfterMethod TearDown" time="0.0" classname="com.rediff.testcases.LoginTest">
    <failure type="com.framework.exception.MyException" message="TestBase -&amp;gt; embedScreenShotIntoReport() - null">
      <![CDATA[com.framework.exception.MyException: TestBase -> embedScreenShotIntoReport() - null
	at com.framework.report.CaptureBrowserScreenShot.embedScreenShotIntoReport(CaptureBrowserScreenShot.java:88)
	at com.framework.testtemplate.TestBase.TearDown(TestBase.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase name="@BeforeMethod LoginTestSetUp" time="0.0" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod LoginTestSetUp -->
  <testcase name="@AfterMethod TearDown" time="0.0" classname="com.rediff.testcases.LoginTest">
    <failure type="com.framework.exception.MyException" message="TestBase -&amp;gt; embedScreenShotIntoReport() - null">
      <![CDATA[com.framework.exception.MyException: TestBase -> embedScreenShotIntoReport() - null
	at com.framework.report.CaptureBrowserScreenShot.embedScreenShotIntoReport(CaptureBrowserScreenShot.java:88)
	at com.framework.testtemplate.TestBase.TearDown(TestBase.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase name="@BeforeMethod LoginTestSetUp" time="0.0" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod LoginTestSetUp -->
  <testcase name="@AfterMethod TearDown" time="0.0" classname="com.rediff.testcases.LoginTest">
    <failure type="com.framework.exception.MyException" message="TestBase -&amp;gt; embedScreenShotIntoReport() - null">
      <![CDATA[com.framework.exception.MyException: TestBase -> embedScreenShotIntoReport() - null
	at com.framework.report.CaptureBrowserScreenShot.embedScreenShotIntoReport(CaptureBrowserScreenShot.java:88)
	at com.framework.testtemplate.TestBase.TearDown(TestBase.java:202)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TearDown -->
  <testcase name="blankUserNameBlankPasswordLoginTest" time="0.009" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- blankUserNameBlankPasswordLoginTest -->
  <testcase name="verifyBrokenLink" time="0.004" classname="com.rediff.testcases.HomePageTest">
    <skipped/>
  </testcase> <!-- verifyBrokenLink -->
  <testcase name="blankUserNameRightPasswordLoginTest" time="0.003" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- blankUserNameRightPasswordLoginTest -->
  <testcase name="blankUserNameWrongPasswordLoginTest" time="0.004" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- blankUserNameWrongPasswordLoginTest -->
  <testcase name="rightUserNameBlankPasswordLoginTest" time="0.009" classname="com.rediff.testcases.LoginTest">
    <failure type="org.testng.TestNGException" message="
The data provider is trying to pass 1 parameters but the method com.rediff.testcases.LoginTest#rightUserNameBlankPasswordLoginTest takes 0">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 1 parameters but the method com.rediff.testcases.LoginTest#rightUserNameBlankPasswordLoginTest takes 0
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1337)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- rightUserNameBlankPasswordLoginTest -->
  <testcase name="rightUserNameWrongPasswordLoginTest" time="0.002" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- rightUserNameWrongPasswordLoginTest -->
  <testcase name="sucessfullLoginTest" time="0.002" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- sucessfullLoginTest -->
  <testcase name="wrongUserNameBlankPasswordLoginTest" time="0.002" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- wrongUserNameBlankPasswordLoginTest -->
  <testcase name="wrongUserNameRightPasswordLoginTest" time="0.002" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- wrongUserNameRightPasswordLoginTest -->
  <testcase name="wrongUserNameWrongPasswordLoginTest" time="0.003" classname="com.rediff.testcases.LoginTest">
    <skipped/>
  </testcase> <!-- wrongUserNameWrongPasswordLoginTest -->
</testsuite> <!-- Rediff WebSite Testing -->
